jira/client.py

```diff
@@ -3600,6 +3600,37 @@ def search_issues(
 
         return issues
 
+    # Archive Issues
+    def archive_issues(self, issues: ResultList[Issue]) -> dict[str, Any]:
+        """Archive a list of Issues.
+
+        Args:
+            issues (ResultsList[Issue]): List of issues to archive
+
+        Returns:
+            Dict[str, Any]: Response from the server indicating success or failure for each issue in the list
```

Comment:
Thank you for your contribution.
Could you provide an example to the end user what this returned dict is?

also dont forget to rebase onto latest main to incorporate some recent fixes